partner_communication/static/src/less/communication_dashboard.less

```diff
@@ -0,0 +1,223 @@
+.o_kanban_view.o_kanban_dashboard.o_partner_communication_kanban {
```

Comment:
The css related to the ribbon is a copy of the one specific to the sales_team dashboard and the one related to the cards is defined in the core module "web" but overrited so the margins are similar to the account_journal dashboard.

Because these stylesheets are not defined as is in core modules this less file is useful. Althought, for future dashboards, we could maybe move this file in the newly created "Compassion Dashboard Modules" and use more generic names for classes ?